### PR TITLE
fix/added-search-button-on-search-box

### PR DESCRIPTION
**Pull request by pkosanam**
Description: added search button on the search input

Prev:
image

Now:
image

@aswanson-nr , request you to kindly confirm if the file changes which I made were the right way to proceed.
Thank you 😊

### CODE REVIEW

**Review by mdumpati — APPROVED** [review]
looks good

File: src/components/IOBanner.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { css } from '@emotion/react';
 import bannerOverlayRight from '../images/io-banner/banner-style-right.svg';
 import bannerOverlayLeft from '../images/io-banner/banner-style-left.svg';
-import { SearchInput } from '@newrelic/gatsby-theme-newrelic';
+import SearchInput from '../@newrelic/gatsby-theme-newrelic/components/SearchInput';
```
Comment (aswanson-nr) [inline src/components/IOBanner.js:6]: I'm not sure we need this change, [Gatsby Shadowing](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/) handles overriding the import for us.

Comment (pkosanam) [reply src/components/IOBanner.js:6]: oh.. got it.. Thank you @aswanson-nr

**Comment by pkosanam** [comment]
I am closing this PR, as the search button functionality is already updated on the newrelic.com/instant-observability website.
Hence, there is no need of this change to be implemented.

**Pull request closed without merging**